lib/rspec/support/spec.rb

```diff
@@ -12,6 +13,7 @@
   c.include RSpecHelpers
   c.include RSpec::Support::WithIsolatedStdErr
   c.include RSpec::Support::FormattingSupport
+  c.include RSpec::Support::EncodingHelpers
```

Comment:
Ok, will remove once we're done discussing. In the meantime, I'm in CST, it's 12:44am and :zzz: